README.md

```diff
@@ -147,3 +147,14 @@ elemeno.getCollectionItem(
   }
 );
 ```
+
+## Caching
+
+The caching module uses the [simple in-memory cache module from ptarjan](https://github.com/ptarjan/node-cache).
+
+To enable it, just call:
+
+#### `elemeno.enableCache([options])`
+
+##### Options
+ - `refreshTime` : Time (in seconds) the cache will delete the stored data (and thus refreshing the data on the next call). Default: 15 mn.
```

Comment:
I was honestly a bit confused at first that the `refreshTime` is accepted in seconds, but the _default_ is listed in minutes. 

I think the execution makes total sense, I just think the comment could be improved a little bit to be something more like:
>Default: 900 seconds (15 min)